enh(readme): add supported versions (#9500)
centreon/centreon: README.md

* Using the [**ISO**](https://docs.centreon.com/current/en/installation/installation-of-a-central-server/using-centreon-iso.html)
* Using the [**VM**](https://docs.centreon.com/current/en/installation/installation-of-a-central-server/using-virtual-machines.html) (tutorial available on [Youtube](https://www.youtube.com/watch?v=rnq9QnljaQE))

<h2> Supported versions <h2>

Regarding the [Products Lifecycle Policy](https://docs.centreon.com/current/en/releases/lifecycle.html),
only the following versions are supported:

* Centreon 20.10.x, released the October 21, 2020, full support
* Centreon 20.04.x, released the April 22, 2020, security and blocking issue support only
* Centreon 19.10.x, released the April 23, 2019, security support only

> 

Reviewer: ```suggestion
If your version is not one of the 3 versions specified above, we recommend that you update your platform immediately.

* centreon 2.8.x, is no longer supported. Specific developments are still made for a specific customer but no package will be provided on our public YUM repository.

```

Author: I don't want to talk about specific support.

* Using the [**ISO**](https://docs.centreon.com/current/en/installation/installation-of-a-central-server/using-centreon-iso.html)
* Using the [**VM**](https://docs.centreon.com/current/en/installation/installation-of-a-central-server/using-virtual-machines.html) (tutorial available on [Youtube](https://www.youtube.com/watch?v=rnq9QnljaQE))

<h2> Supported versions <h2>

Regarding the [Products Lifecycle Policy](https://docs.centreon.com/current/en/releases/lifecycle.html),
only the following versions are supported:

> 

Reviewer 2: ```suggestion
* Centreon 21.04.x, released on April 21, 2021, full support
```

> * Centreon 20.10.x, released the October 21, 2020, full support

Reviewer 2: ```suggestion
* Centreon 20.10.x, released on October 21, 2020, security and blocking issue support only
```

> * Centreon 20.04.x, released the April 22, 2020, security and blocking issue support only

Reviewer 2: ```suggestion
* Centreon 20.04.x, released on April 22, 2020, security support only
```

> * Centreon 19.10.x, released the April 23, 2019, security support only

Reviewer 2: ```suggestion
```
